hw/ip/edn/data/edn.hjson

```diff
@@ -84,24 +84,23 @@
                 software before any further commands can be issued to these FIFOs.
                 '''
         }
-        { bits: "2",
-          name: "AUTO_REQ_MODE",
+        { bits: "3:2",
```

Comment:
I recommend to do a proper enum here:
```
{ bits: "3:2",
  name: "HW_REQ_MODE",
  resval: ?
  desc: General description of the field, here I would list what happens if an invalid value is selected.
  enum: [
    { value: 0,
      name: "DISABLE"
      desc: mode specific description
    },
    { value: 1,
      name: "AUTO_REQ_MODE",
      desc: mode specific description
    },
    { value: "2",
      name: "BOOT_REQ_MODE",
      desc:
    },
    // No need to list the reserved value.
    // The regtool automatically prints a "Other values are reserved" statement.
  ]
```
This will look as follows in the online documentation: https://docs.opentitan.org/hw/ip/aes/doc/#Reg_ctrl_shadowed

Reply:
This is a much better format, I will change to this. Thanks @vogelpi for you review feedback!

Reply:
You're welcome @mwbranstad , thanks for making the propsed changes :+1: